index.md

```diff
@@ -14,19 +14,19 @@ The NHS England Digitial Analytics and Research Team *(formerly NHSX Analytics U
-- how to collate and Curate NHS-related **text data** sources into a training corpus which represents how NHS clinicains and patients talk about their health
-- demonstration of system and patient **simulation** methods such as disrecete event simulation and agent based modelling 
-- how to define and deal with privacy concerns in text data and remote analysis using **privacy enhancing technologies**
-- methods for extracting insight from large scale event logs and multi-modal data using **graph-based** approaches
-- establishing **open ways of working** and supporting reproducible analytical pipelines across NHS analysts
-- applied **forecasting** with explainability
+Our research aims to address some of these issues by building data foundations to enable innovation in the near future. The first of these foundations is to **increase the understanding of the appropriate representation of structured and unstructured data**.  When multiple points of healthcare data and/or different modalities of data are linked it becomes complex with both intra and inter correlations, unknown confounding variables and often poor quality.  The complexity often makes it impossible to deal with directly in an efficient manner and the variable relationships and quality make causal insights difficult to identify.  To deal with these issues we need to have methods for representing this data in appropriate formats which can then be analysed or used in a model.  We've started to explore this area through understanding the appropriate application of large-scale language models, exploring multi-modal representations of text and images together and graph representations of data to capture entity relationships rather than ordered correlations.
 
-Within the DART, the innovation branch attempts to push forwards these research interests through our PhD Data Science Internship Scheme as well as internal projects to connect academia with NHS operations. 
+The second pillar is centred around our belief that unstructured data (especially text data) has huge unlocked potential for healthcare analysis.  A key blocker to using unstructured text data is the potential sensitivity within the data.  Often this is partially dealt with by anonymising for patient identifiable information but this can still leave content which is contextual or domain specific and therefore still disclosive.  We are therefore aiming to **highlight the privacy of unstructured data beyond PII** through understanding the problem and currently available options and then creating tool to calculate a privacy fingerprint for a data base or piece of text.  This fingerprint would be a structured way of addressing the risk of disclosure which could then be used in an IG process to understand what masking/anonymisation is required or what limited terms of use need to be applied to the data.  By having a tangible way of addressing this issue we aim to increase the sharing of these data with confidence. 
```

Comment:
Possible link to the `Functional Anonymisation Paper` (e.g. https://eprints.soton.ac.uk/417832/1/Functional_Anonymisation_and_the_Data_Environment_Final.pdf) which gives a nice overview of challenges with anonymisation in general?